unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/SceneController.cs

```diff
@@ -495,6 +495,78 @@ public void SendSceneReady(string sceneId)
             OnReadyScene?.Invoke(sceneId);
         }
 
+        public bool TryGetScene(string id, out ParcelScene scene)
```

Comment:
Please move all those methods to `WorldState` as they should belong there, they were moved as part of our ongoing refactor.